config/software/libyaml-windows.rb

```diff
@@ -43,8 +43,8 @@
   command "7z.exe x #{project_file} -o#{tmpdir} -r -y", env: env
 
   # Now extract the files out of tar archive.
-  command "7z.exe x #{File.join(tmpdir, "libyaml-0.1.6-x86-windows.tar")} -o#{tmpdir} -r -y", env: env
+  command "7z.exe x #{File.join(tmpdir, 'libyaml-0.1.6-x86-windows.tar')} -o#{tmpdir} -r -y", env: env
 
   # Now copy over libyaml-0-2.dll to the build dir
-  copy "#{tmpdir}/bin/libyaml-0-2.dll", "#{install_dir}/embedded/bin/libyaml-0-2.dll"
+  copy '#{tmpdir}/bin/libyaml-0-2.dll', '#{install_dir}/embedded/bin/libyaml-0-2.dll'
```

Comment:
Which is why the guidance to use single quotes when you don't _yet_ need
interpolation creates bugs :)

On Tue, Apr 14, 2015 at 6:18 PM, Seth Chisamore <notifications@github.com>
wrote:

> In config/software/libyaml-windows.rb
> <https://github.com/chef/omnibus-software/pull/396#discussion_r28386853>:
>
> >
> >    # Now copy over libyaml-0-2.dll to the build dir
> > -  copy "#{tmpdir}/bin/libyaml-0-2.dll", "#{install_dir}/embedded/bin/libyaml-0-2.dll"
> > +  copy '#{tmpdir}/bin/libyaml-0-2.dll', '#{install_dir}/embedded/bin/libyaml-0-2.dll'
>
> This needs double quotes here or the string interpolation won't work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/omnibus-software/pull/396/files#r28386853>.
>